config/services.yaml

```diff
@@ -122,6 +122,11 @@ services:
             $genesisTime: '%env(GENESIS_TIME)%'
             $blockSeqTime: '%env(BLOCK_SEQ_TIME)%'
 
+    Adshares\AdsOperator\GuzzleHttp\ApiClient:
```

Comment:
This should be a dev service only

Reply:
I've created a new configuration (DI) file for `test` environment